lib/rdf/serialization.ts

```diff
@@ -40,298 +41,305 @@ const xsdUnsignedShort = xsd + 'unsignedShort';
 const xsdUnsignedByte = xsd + 'unsignedByte';
 const xsdPositiveInteger = xsd + 'positiveInteger';
 
-export const exportLiteralTerm = (term: string, dataFactory: DataFactory): Literal => {
-  const [, encoding, datatype, language, value] = term.split('\u0001');
-  switch (datatype) {
-    case rdfLangString:
-      if (language !== '') {
-        return dataFactory.literal(value, language);
-      }
-      return dataFactory.literal(value);
-    default:
-      return dataFactory.literal(value, dataFactory.namedNode(datatype));
+export class RdfSerialization {
+  constructor(readonly prefixes: TSRdfPrefixes) {
```

Comment:
I dislike this syntax as it hides the assignment of `prefix` to the `RdfSerialization` instance. For newcomers to TS it can get confusing - could you make this assignment explicit?